docs/USE-CASES.adoc

```diff
@@ -0,0 +1,35 @@
+= _Essentials_ Use Cases
+:toc:
+:sectnums:
+
+This document summarizes the Priority 1 use cases for _Jenkins Essentials_.
+
+NOTE: This is still early work.
+To help the thought process, the cases definition and analysis was done by thinking about what a demonstration would need to go through.
```

Comment:
This has been going for quite long. So as there is this notice, going to merge it as at least the core intent of this (the use cases themselves, not their risks all analyzed) seems to be fulfilled.